pre-docs/bolt_installing.md

```diff
@@ -241,10 +241,3 @@ To disable the collection of analytics data add the following line to `~/.puppe
 ```
 disabled: true
 ```
-
-## Using Kerberos over SSH
```

Comment:
I think this is also linked from [pre-docs/index.md] - does that automatically get updated?